docs/source/user_guides/basic_conceptions.rst

```diff
@@ -0,0 +1,303 @@
+            },
+            "greeting_node": {
+                RESPONSE: Message(text="Hi! Let's play ping-pong!"),
+                TRANSITIONS: {
+                    lbl.forward(): cnd.regexp(r"^[P|p]ing!?$"),
+                },
+            },
+            "ping_pong_node": {
+                RESPONSE: Message(text="Pong!"),
+                TRANSITIONS: {
+                    lbl.repeat(): cnd.regexp(r"^[P|p]ing!?$"),
+                },
+            },
+            "fallback_node": {
+                RESPONSE: fallback_response,
+                TRANSITIONS: {
+                    "greeting_node": cnd.true(),
+                },
+            },
+        },
+    }
+
+    pipeline = Pipeline.from_script(
+        ping_pong_script,
+        start_label=("ping_pong_flow", "start_node"),
+        fallback_label=("ping_pong_flow", "fallback_node"),
+    )
+
+    if __name__ == "__main__":
+        pipeline.run()
+
+That's what we've changed:
+
+* ``greeting_node`` and ``response_node`` were merged, the resulting ``ping_pong_node`` has a
+  callback response, it proposes user to play ping-pong if the previous node was ``start_node`` and
+  plays ping-pong otherwise.
+
+* ``fallback_node`` has a callback response as well, it prints different messages according to the
+  previous node.
+
+.. note::
+
+    See `tutorial on response functions`_.
+
+* A special function ``get_previous_node_name`` was written to determine the name of the previous
+  visited node. It utilizes ``labels`` attribute of the ``Context`` object.
+
+.. note::
+
+    See `documentation of Context object`_.
+
+* Transitions were changed: transitions to next, previous and current node were replaced with special
+  standard transitions.
+
+.. note::
+
+    See `tutorial on transitions`_.
+
+* Conditions were changed: now regular expressions are used to check user text input value.
+
+.. note::
+
+    See `tutorial on conditions`_.
+
+Further exploration
+~~~~~~~~~~~~~~~~~~~
+
+There are still come capabilities of Dialog Flow Framework that remain uncovered by this tutorial.
+
+For example:
+
+* You can use ``GLOBAL`` transitions that will be available from every node in your script.
+  See `tutorial on global transitions`_.
+
+* You can serialize context (available on every transition and response)
+  to json or dictionary in order to debug it or extract some values.
+  See `tutorial on context serialization`_.
+
+* You can alter user input and modify generated responses.
+  User input can be altered with ``PRE_RESPONSE_PROCESSING`` and will happen **before** response generation.
+  See `tutorial on pre-response processing`_.
+  Node response can be modified with ``PRE_TRANSITION_PROCESSING`` and will happen **after** response generation.
+  See `tutorial on pre-transition processing`_.
+
+* Additional data ``MISC`` can be added to every node, flow and script itself.
+  Warning! Unlike the other Context fields, the MISC values are not shared across script executions.
```

Comment:
I don't understand what this is about. The text mentions MISC as a context field but the linked tutorial is about script MISC.
Am I missing something and they are the same?

I also don't understand what is meant by "sharing values between script executions" (I don't see how this relates to either script misc or context misc).

Reply:
I might've well missed something, I've never used script MISC.
Would you have any suggestions about how should we mention script MISC in this guide (i.e. what should we write about them)?

Reply:
Script MiSC is already mentioned (script MISC is the one that is stored inside nodes, flows and script; and the linked tutorial is about it).

It's the second sentence that is confusing me (script MISC is not a context field, and I don't understand what the second part of the sentence is supposed to mean (none of the changes services make to script nodes during execution are persistent)).

Reply:
Removed the second sentence